lib/constructs/AgentDefinitionBuilder.ts

```diff
@@ -209,6 +209,35 @@ export class AgentDefinitionBuilder extends Construct {
         return this;
     }
 
+    /**
+     * To allow agent to generate, run, and troubleshoot your application code in a secure test environment.
+     * We need to add an action group with the parentActionGroupSignature 
+     * field set to AMAZON.CodeInterpreter.
+     * @returns AgentDefinitionBuilder object with a new action set.
+     */
+    public withCodeInterpreter() {
+        const codeInterpreterAction: bedrock.CfnAgent.AgentActionGroupProperty = {
+            actionGroupName: CODE_INTERPRETER_ACTION_NAME,
```

Comment:
Please take this as an optional input from the user. `withCodeInterpreter(actionName: string)`

Reply:
Not sure this is required or exactly what you mean. Is this a blocker?